status/backend.json

```diff
@@ -25,6 +25,18 @@
         "en-EN": "Our systems may respond slowly, we apologize for the inconvenience."
       }
     },
+    "cashback": {
```

Comment:
you are duplicating the `"cashback"` entry
Maybe we should remove the same entry at line20